pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -349,10 +352,19 @@ func (a Adapter) execDevfile(commandsMap common.PushCommandsMap, componentExists
 		// Get Init Command
 		command, ok := commandsMap[versionsCommon.InitCommandGroupType]
 		if ok {
-			compInfo.ContainerName = command.Exec.Component
-			err = exec.ExecuteDevfileBuildAction(&a.Client, *command.Exec, command.Exec.Id, compInfo, show, a.machineEventLogger)
-			if err != nil {
-				return err
+			if command.Composite != nil {
+				fmt.Println("Composite detected")
+				//err = a.execCompositeCommand(command.Composite, compInfo, show, podName)
```

Comment:
comment found

Reply:
The println can be removed too, will fix that 😄